Wurstpack/wurstscript/lib/data structures/LinkedList.wurst

```diff
@@ -151,8 +151,8 @@ public class LLIterator<Q>
 		if current != dummy
 			current.next.prev = current.prev
 			current.prev.next = current.next
-			current = current.prev
 			destroy current
+			current = current.prev
```

Comment:
better than a bug ;) i fixed it in stdlib2 the same way some time ago.
I hope
````
			let temp = current
			current = current.prev
			destroy temp
````
gets inlined then